src/common/interfaces/common.interface.ts

```diff
@@ -40,6 +41,12 @@ export interface ILimitsFilter {
   order: string;
 }
 
+export class ILimitsFilterDto implements ILimitsFilter {
```

Comment:
As I see this ILimitsFilterDto is completly the same as ILimitsFilter. Do we really need two different representations or we can have just one?

Reply:
You are correct!!!
I'm going to remove it.